endpoint/app.py

```diff
@@ -69,15 +68,45 @@
     }
 })
 
+access = '80658bec0ce34880a9bbc3f3f50ca60b'
+secret = '9069592f9b314a52b26915e35fbef581'
+
+
+def add_object(access_key,secret_key, bucketname, prefix):
+    host = 'https://cf2.cloudferro.com:8080'
+    s3 = boto3.resource('s3',aws_access_key_id=access_key, aws_secret_access_key=secret_key, endpoint_url=host,)
+
+    bucket =s3.Bucket(bucketname)
 
+    bucket.put_object(Key=prefix)
 
-@application.route('/userinfo', methods=['POST'])
+def lookup_objects(access_key, secret_key, bucketname, pref):
+    host = 'https://cf2.cloudferro.com:8080'
+    s3 = boto3.resource('s3',aws_access_key_id=access_key, aws_secret_access_key=secret_key, endpoint_url=host,)
+
+    bucket=s3.Bucket(bucketname)
+    for obj in bucket.objects.filter():
+        if pref == obj.key:
+            return True
+        else: 
+            return False
+
+@application.route('/userinfo', methods=['GET'])
 def store():
```

Comment:
The function name `store` is misleading. Usually, the name can be the same as the route, so `def userinfo(...):` should be fine.